src/components/our-vision-story-blocks.js

```diff
@@ -24,14 +25,17 @@ const OurVisionStoryBlocks = () => {
       }
     }
   `)
+  const intl = useIntl()
   const storyData = storysQuery.contentJson.expanded_stories
   return (
     <div className="our-vision-text">
       {storyData.map((node, index) => (
         <StoryBlock
           imageSource={node.image.childImageSharp.fluid}
           altText={node.altText && node.altText}
-          description={node.description && node.description}
+          description={intl.formatMessage({
```

Comment:
It's annoying, but you know these 'fail' when the literal text ends up on the site instead of the actual content. So we can know this one didn't work. 

This one isn't really your fault though; in json files, this content is is actually called ```expanded_stories```

Apologies for the confusing naming; but you can see on line 29 where the object is called expanded_stories. You can also check that in the graphql query above and double-check in en.json if you're unsure. 

For the LAST gotcha, in the content these keys are separated by hypens, not underscores. It's some silly JSON vs GraphQL naming convention. 

So this is basically the rule: 

If the 'name' of the key is more than two works, in GraphQL those values are separated by underscores. But in the JSON file they will be separated by hyphens. So if something isn't popping up, try that first.

Reply:
So, I am getting the same issues trying to change the names from underscores to hyphens. On save it adds spaces to the dashes and fails to build.